api/wsclient.py

```diff
@@ -38,7 +38,7 @@ def onMessage(self, payload, isBinary):
 
 if __name__ == '__main__':
 
-    factory = WebSocketClientFactory("ws://localhost:18466")
+    factory = WebSocketClientFactory("ws://seed.openbazaar.org:18466")
```

Comment:
will this URL:port somehow be configurable in the future?
(say to avoid firewalls)